apps/crn-frontend/src/analytics/collaboration/TeamCollaboration.tsx

```diff
@@ -0,0 +1,259 @@
+  }
+  return data.map((row) => ({
+    id: row.id,
+    name: row.name,
+    isInactive: row.isInactive,
+    ...row.OutputsCoProducedAcross,
+    type: 'across-teams',
+  }));
+};
+
+const TeamCollaboration: React.FC<CollaborationProps> = ({ type }) => {
+  const { currentPage, pageSize } = usePaginationParams();
+
+  const teamItems: TeamCollaborationResponse[] = [
+    {
+      id: '1',
+      name: 'Team A',
+      isInactive: true,
+      OutputsCoProducedWithin: {
+        Article: 1,
+        Bioinformatics: 1,
+        Dataset: 0,
+        'Lab Resource': 1,
+        Protocol: 0,
+      },
+      OutputsCoProducedAcross: {
+        Article: 1,
+        Bioinformatics: 2,
+        Dataset: 1,
+        'Lab Resource': 0,
+        Protocol: 2,
+        'Collaboration Details': [
+          {
+            id: '2',
+            name: 'Team B',
+            isInactive: false,
+            Article: 1,
+            Bioinformatics: 2,
+            Dataset: 1,
+            'Lab Resource': 0,
+            Protocol: 2,
+          },
+        ],
+      },
+    },
+    {
+      id: '2',
+      name: 'Team B',
+      isInactive: false,
+      OutputsCoProducedWithin: {
+        Article: 1,
+        Bioinformatics: 2,
+        Dataset: 3,
+        'Lab Resource': 4,
+        Protocol: 5,
+      },
+      OutputsCoProducedAcross: {
+        Article: 1,
+        Bioinformatics: 2,
+        Dataset: 2,
+        'Lab Resource': 1,
+        Protocol: 2,
+        'Collaboration Details': [
+          {
+            id: '1',
+            name: 'Team A',
+            isInactive: true,
+            Article: 1,
+            Bioinformatics: 2,
+            Dataset: 1,
+            'Lab Resource': 0,
+            Protocol: 2,
+          },
+          {
+            id: '5',
+            name: 'Team E',
+            isInactive: false,
+            Article: 0,
+            Bioinformatics: 0,
+            Dataset: 1,
+            'Lab Resource': 1,
+            Protocol: 0,
+          },
+          {
+            id: '6',
+            name: 'Team F',
+            isInactive: false,
+            Article: 0,
+            Bioinformatics: 0,
+            Dataset: 1,
+            'Lab Resource': 0,
+            Protocol: 0,
+          },
+        ],
+      },
+    },
+    {
+      id: '3',
+      name: 'Team C',
+      isInactive: false,
+      OutputsCoProducedWithin: {
+        Article: 1,
+        Bioinformatics: 0,
+        Dataset: 3,
+        'Lab Resource': 0,
+        Protocol: 0,
+      },
+      OutputsCoProducedAcross: {
+        Article: 0,
+        Bioinformatics: 0,
+        Dataset: 0,
+        'Lab Resource': 0,
+        Protocol: 0,
+        'Collaboration Details': [],
+      },
+    },
+    {
+      id: '4',
+      name: 'Team D',
+      isInactive: false,
+      OutputsCoProducedWithin: {
+        Article: 0,
+        Bioinformatics: 0,
+        Dataset: 0,
+        'Lab Resource': 0,
+        Protocol: 0,
+      },
+      OutputsCoProducedAcross: {
```

Comment:
mixing output types and collaboration details seems a bit confusing. 
Could this be
```
{OutputsCoproducedAcross:{
  Types: {
    Article: 0,
    ...
    },
    CollaborationDetails: []
   }

Reply:
I could group them into 'byDocumentType' and 'byTeam'
i.e `{ OutputsCoProducedAcross: {
         byDocumentType: {
           Article: 0,
           ...
           },
         byTeam: []`} How does that sound?